docs/content/docs/8.deployment/3.vercel.md

```
@@ -0,0 +1,118 @@
+| Variable | Example |
+| -------- | ------- |
+| `SITE_URL` | `https://your-app.vercel.app` or your custom domain |
+| `BETTER_AUTH_SECRET` | generated secret |
+| `GITHUB_CLIENT_ID` | only if GitHub auth is enabled |
+| `GITHUB_CLIENT_SECRET` | only if GitHub auth is enabled |
+
+If you use static JWKS, also set:
+
+| Variable | Example |
+| -------- | ------- |
+| `JWKS` | output from `npx convex run auth:getLatestJwks | npx convex env set JWKS` |
```

markdownlint-cli2 (0.21.0)

[warning] Table column count
Expected: 2; Actual: 3; Too many cells, extra data will be missing

(MD056, table-column-count)